src/download.rs

```diff
@@ -282,6 +282,15 @@ pub fn download_file_and_unpack(download_cfg: &DownloadCfg, dst_dir_path: &Path)
     Ok(())
 }
 
+pub fn link_to_toolchain(toolchain_dir: PathBuf, bins: Vec<PathBuf>) -> Result<()> {
```

Comment:
```suggestion
pub fn link_to_toolchain(toolchain_dir: &Path, bins: Vec<PathBuf>) -> Result<()> {
```

`toolchain_dir` is never used as a `PathBuf` in this function